test-positive/mermaid.md

```diff
@@ -138,4 +138,30 @@ stateDiagram
     Choose --> Still
     Choose --> Crash
     Crash --> [*]
-```
\ No newline at end of file
+```
+10. indented.mmd
+    ```mermaid
+    stateDiagram
+        state Choose <<fork>>
+        [*] --> Still
+        Still --> [*]
+
+        Still --> Moving
+        Moving --> Choose
+        Choose --> Still
+        Choose --> Crash
+        Crash --> [*]
+    ```
+11. space-appended.mmd
+```mermaid
+stateDiagram
+    state Choose <<fork>>
+    [*] --> Still
+    Still --> [*]
+
+    Still --> Moving
+    Moving --> Choose
+    Choose --> Still
+    Choose --> Crash
+    Crash --> [*]
+```   
```

Comment:
It might be worth adding a comment to make these spaces more visible.

I think most of my text editors are set up to auto-delete trailing spaces, so a comment might help me avoid accidentally deleting them :)

````suggestion
%% ↓↓↓↓↓↓↓↓↓ should still find mermaid code even with trailing spaces after the ```
```         
````

Reply:
At the time, when I configured Percy, there was no limitation for open source projects. Things might have changed since then.
A comment as suggested by @aloisklink would be appreciated.

Reply:
I'm already on it to reduce the amount of percy images again as mentioned

Reply:
> @aloisklink I added the comment and implemented a check if the test case 9. is in the state that is required.

Awesome :) Not required, but definitely appreciated! Thanks for the PR. :rocket: